### DISCUSSION TITLE
JD Builder on Astroid

### DISCUSSION THREAD

**Discussion by eagleshout** [templaza/astroid-framework#204] [Q&A] [answered]
Is there any intent reboot what was JD Builder? Ever since installing this updated version of Astroid the JD Builder is no longer accessible. If NOT, are there any replacement page builders compatible with previously built pages?

**Comment by Chacapamac** [comment, marked as answer]
The support for **Joomdev commercial templates** and **Extensions** was taking over by **Stackideas** —> https://stackideas.com

**Maybe, they can help you?** 

**_Please, come back here to tell Us, how it went?_**

Good Weekend.

This is one problem about these “Builders” they all work their own ways, with their own codes and are not really compatible or be replaced. 
I personally test all of the big names and went back to JCE as you can have the same WYSIWYG editor experience that are always compatible and use all the core Joomla power of edition. With the great possibility of premade custom templates, Custom Styles, easy use of Bootstrap columns and the full power of JCE, no need of any other editor in Joomla and can be make really easy to use for clients.

**Hint:** JCE is working right now to add the ability to drag & drop any block elements in the editor itself. In fact you can already do the same with JCE Templates where you can just drop premade custom sections anywhere.
